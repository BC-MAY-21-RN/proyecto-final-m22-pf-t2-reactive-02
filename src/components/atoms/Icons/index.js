import React from 'react';
import {View} from 'react-native';

export default function Icons({IconProp, styles}) {
  return (
    <View style={styles}>
      <IconProp />
    </View>
  );
}
