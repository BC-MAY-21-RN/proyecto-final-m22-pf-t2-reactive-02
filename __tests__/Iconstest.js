import 'react-native';
import React from 'react';
import {View} from 'react-native';
import renderer from 'react-test-renderer';
import Icons from '../src/components/atoms/Icons';

describe('Test from Icons component', () => {
  it('Icons snapshot test', () => {
    const renderIcons = renderer
      .create(<Icons IconProp={View} styles={{}} />)
      .toJSON();
    expect(renderIcons).toMatchSnapshot();
  });
});
