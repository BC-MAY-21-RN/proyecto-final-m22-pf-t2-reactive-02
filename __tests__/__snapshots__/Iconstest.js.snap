// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from Icons component Icons snapshot test 1`] = `
<View
  style={Object {}}
>
  <View />
</View>
`;
